hivemq-edge/src/frontend/src/modules/Bridges/components/panels/SubscriptionsPanel.spec.cy.tsx

```diff
@@ -65,13 +65,16 @@ describe('SubscriptionsPanel', () => {
     cy.percySnapshot('Component: SubscriptionsPanel')
   })
 
-  it('should initialise with OpenAPI defaults', () => {
+  it.only('should initialise with OpenAPI defaults', () => {
```

Comment:
question(non-blocking): guess all tests should run but only this once will be executed once merged

```suggestion
  it('should initialise with OpenAPI defaults', () => {
```